src/shims/foreign_items.rs

```diff
@@ -1008,6 +1008,13 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             .read_scalar(tcx, errno_ptr, Size::from_bits(32))?
             .not_undef()
     }
+
+    fn consume_io_error(&mut self, e: std::io::Error) -> InterpResult<'tcx> {
+        self.eval_context_mut().set_last_error(Scalar::from_int(
+                e.raw_os_error().unwrap(),
+                Size::from_bits(32),
```

Comment:
rustfmt didn't run here? :laughing:

Reply:
Hah, probably it didn't. I have autoformatting disabled in all the rust-lang related projects to avoid creating huge diffs :P